tests/util/bft.py

```diff
@@ -393,6 +393,46 @@ def force_quorum_including_replica(self, replica_id, primary=0):
                     nursery.start_soon(self._assert_state_transfer_not_started,
                                        r)
 
+    async def assert_fast_path_prevalent(self, as_of_seq_num):
+        metric_key = ['replica', 'Counters', 'slowPathCount']
+        nb_slow_paths = await self.metrics.get(0, *metric_key)
+
+        assert nb_slow_paths - as_of_seq_num <= 1, \
+            f'Fast path is not prevalent for n={self.config.n}, f={self.config.f}, c={self.config.c}.'
+
+    async def assert_slow_path_prevalent(self, as_of_seq_num):
+        metric_key = ['replica', 'Gauges', 'lastExecutedSeqNum']
+        all_executed_sequences = await self.metrics.get(0, *metric_key)
+
+        metric_key = ['replica', 'Counters', 'slowPathCount']
+        nb_slow_paths = await self.metrics.get(0, *metric_key)
+
+        assert nb_slow_paths >= all_executed_sequences - as_of_seq_num, \
```

Comment:
I think I get what's going on here, but a doc comment for the method would be useful.

Reply:
Agreed - both the implementation and the doc need improvement.